control/mpc_follower/config/mpc_follower.param.yaml

```diff
@@ -6,8 +6,8 @@
     traj_resample_dist: 0.1         # path resampling interval [m]
     enable_yaw_recalculation: false # flag for recalculation of yaw angle after resampling
     use_steer_prediction: false     # flag for using steer prediction (do not use steer measurement)
-    admisible_position_error: 5.0   # stop mpc calculation when error is larger than the following value
-    admisible_yaw_error_: 1.57      # stop mpc calculation when error is larger than the following value
+    admissible_position_error: 5.0  # stop mpc calculation when error is larger than the following value
+    admissible_yaw_error_: 1.57     # stop mpc calculation when error is larger than the following value
```

Comment:
Probably should remove `_`?

Reply:
Or, should be `admissible_yaw_error_deg` as written in README.md?

Reply:
@kmiya Ah yes, to make the unit explicit is better!
However, `1.57` is a bit strange value for degrees because it's `pi / 2`.
Could you check the code and confirm the correct unit?
Or @TakaHoribe can you answer?

Reply:
sorry, the readme is old. `rad` is the correct unit. I'll fix the document later.

Reply:
@kmiya So then, let's use `admissible_yaw_error_rad` in code and document!

Reply:
Have been changed in the commit https://github.com/tier4/AutowareArchitectureProposal.iv/pull/428/commits/3dea05674d3b26a760c6357006701da4398f75b1 .